peerconnection.go

```diff
@@ -1773,13 +1769,11 @@ func (pc *PeerConnection) Close() error {
 	}
 
 	// https://www.w3.org/TR/webrtc/#dom-rtcpeerconnection-close (step #5)
-	if pc.sctpTransport != nil {
```

Comment:
`sctpTransport` is never nil. If something is depending on this behavior I would like to know